.github/workflows/periodic-update.yml

```diff
@@ -1,10 +1,10 @@
-name: Update Validators Info
+name: Periodic Update
 on:
   schedule:
-    # Run once per day at 19:00 UTC or 12PM PST
-    - cron:  '0 19 * * *'
+    # Run twice per day (at 7:00UTC/12amPST, 19:00UTC/12pmPST)
+    - cron:  '0 7,19 * * *'
 jobs:
-  update-validators:
+  periodic-update:
```

Comment:
`periodic-update` you already have a name for this workflow. job should describe what it does